ebos/eclgenericthresholdpressure.cc

```diff
@@ -220,6 +220,34 @@ configureThpresft_()
     }
 }
 
+template<class Grid, class GridView, class ElementMapper, class Scalar>
+std::vector<Scalar>
+EclGenericThresholdPressure<Grid,GridView,ElementMapper,Scalar>::
+getRestartVector() const
+{
+    if (!enableThresholdPressure_)
+        return {};
```

Comment:
Might the size 0 of the vector be a problem?

Reply:
no. the writer also checks if this is enabled, https://github.com/OPM/opm-simulators/blob/master/ebos/eclgenericwriter.cc#L466